detection_rules/etc/version.lock.json

```diff
@@ -1977,10 +1996,20 @@
     "version": 8
   },
   "3a657da0-1df2-11ef-a327-f661ea17fbcc": {
+    "min_stack_version": "8.13",
+    "previous": {
+      "8.10": {
+        "max_allowable_version": 102,
+        "rule_name": "Rapid7 Threat Command CVEs Correlation",
+        "sha256": "23e49f0f8d57d3b70852d1ff51fde7a12744141f9986f4fa048aba19f7db89a1",
+        "type": "threat_match",
+        "version": 3
+      }
+    },
     "rule_name": "Rapid7 Threat Command CVEs Correlation",
     "sha256": "84bf983155b5e76077e32a0adf47cc76be94453dbd39a996d7cb55b112a6eb99",
     "type": "threat_match",
-    "version": 2
+    "version": 103
```

Comment:
Is this version bump and issue?
![image](https://github.com/user-attachments/assets/b0fc8370-3d30-4967-87c2-f962caf92b3a)

Reply:
We have fixed the versions here https://github.com/elastic/detection-rules/commit/2ee5ae1f197c109f09bc23be7fa9f251ff91871f
 
 Please see min-stack here https://github.com/elastic/detection-rules/pull/3961/files#diff-d539b5f6ddb25f5e1ed43553241cee8ba802cd4417097d8fafcb2f90b557b935R1999

Reply:
Cool thanks!